### ISSUE TITLE
Is it compatible with Bootstrap 5?

### ISSUE THREAD

**Issue by Mani1124** [Johann-S/bs-stepper#290]

**Comment by tienomeis** [comment]
Yes, i use it with Bootstrap 5. This stepper is not tied to Bootstrap 4 since it is based on javascript.

Apart from that the repository does not seem to get much attention lately, so you will have to do fixes and adaptations on your own.
